src/main/java/org/gridsuite/network/map/dto/mapper/TieLineInfosMapper.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.network.map.dto.mapper;
+
+import com.powsybl.iidm.network.*;
+import org.gridsuite.network.map.dto.ElementInfos;
+import org.gridsuite.network.map.dto.definition.tieline.TieLineMapInfos;
+import org.gridsuite.network.map.dto.utils.ElementUtils;
+
+import static org.gridsuite.network.map.dto.utils.ElementUtils.*;
+
+/**
+ * @author Rehili Ghazwa <ghazwa.rehili at rte-france.com>
+ */
+public final class TieLineInfosMapper {
+    private TieLineInfosMapper() {
+    }
+
+    public static ElementInfos toData(Identifiable<?> identifiable, ElementInfos.ElementInfoType dataType) {
+        switch (dataType.getInfoType()) {
+            case MAP:
+                return toMapInfos(identifiable, dataType.getDcPowerFactor());
+            default:
+                throw new UnsupportedOperationException("TODO");
```

Comment:
Some more information that simply "TODO" would be useful. At least something like "TODO : TieLineInfosMapper toData exception". It would make the tracing much easier.